project_proposal/tex/front_page.tex

```diff
@@ -1,47 +1,47 @@
-\begin{titlepage}
-\begin{center}
-% Title
-{ \LARGE \bfseries Bierproductie \\[0.4cm]}
-A management system for brewing machines
-\begin{figure}[H]
-\centering 
-\includegraphics[scale=0.5]{images/system_drawing.png}
-\label{figure:bierproductie_system}
-\end{figure}
-
-Softwareteknologi\\
-\vspace{2mm}
-Semesterproject 3. semester, ST3-PRO\\
-\vspace{2mm}
-\textbf{Project Period:} 31.08.2020 - 19.12.2020 \\
-\vspace{2mm}
-\textbf{Hand in date:} 19.12.2020 \\
-
-\vspace{7mm}
-
-\textbf{Group 06:} \\
-\vspace{2mm}
-Jakob Rasmussen, jakra19@student.sdu.dk \\
-\vspace{2mm}
-Kenneth M. Christiansen kechr19@student.sdu.dk \\
-\vspace{2mm}
-Kevin K. M. Petersen, kepet19@student.sdu.dk \\
-\vspace{2mm}
-Kristian N. Jakobsen, kjako19@student.sdu.dk \\
-\vspace{2mm}
-Simon Jørgensen, sijo819@student.sdu.dk \\
-
-\vspace{7mm}
-
-\textbf{Supervisor:} Parisa Niloofar, parni@mmmi.sdu.dk \\
-
-% Bottom of page
-\vfill
-
-University of Southern Denmark \\
-The Faculty of Engineering \\
-The Mærsk Mc-Kinney Møller Institute \\
-Campusvej 55, 5230 Odense M 
-
-\end{center}
+\begin{titlepage}
+\begin{center}
+% Title
+{ \LARGE \bfseries Bierproductie \\[0.4cm]}
+A management system for brewing machines
+\begin{figure}[H]
+\centering 
+\includegraphics[scale=0.5]{images/system_drawing.png}
+\label{figure:bierproductie_system}
+\end{figure}
+
+Softwareteknologi\\
```

Comment:
Found the proper English name: Bachelor of Engineering, Software Technology

Reply:
fix next commit